### DISCUSSION TITLE
azure connection string to identify

### DISCUSSION THREAD

**Discussion by georgestod** [delta-io/delta-rs#3245] [Q&A] [answered]
Hi,
It seems to me the 'AZURE_STORAGE_CONNECTION_STRING' storage option to load a table is not implemented yet, am I correct?

**Comment by roeap** [comment, marked as answer]
that is correct - connection string parsing requires quite a bit of logic, and we do not use the official SDKs to avoid dependency bloat. If you do have the connection string, extracting valid credentials should be straight forward. assuming the credentials encoded is supported...

**Comment by georgestod** [comment]
ok, thanks a lot !
